README.md

```diff
@@ -31,29 +34,17 @@ If you prefer to develop inside a container with VS Code then do the following s
 3. Click the blue or green rectangle in the bottom left of VS code (should say something like `><` or `>< WSL`). Options should appear in the top center of your screen. Select `Reopen in Container`.
 
 
+### Project Pipeline
 
+1. Collect state's finance campaign data either from web scraping (AZ, MI, PA) or direct download (MN)
+2. User can  go to [this shared Google Drive]('https://drive.google.com/drive/u/2/folders/1HUbOU0KRZy85mep2SHMU48qUQ1ZOSNce') to download each state's data to their local repo following this format: repo_root / "data" / "raw" / <State Initial> / "file"
+3. Open in development container which installs all necessary packages.
+4. Use utils/pipeline.py to preprocess, clean, standardize, and create tables for each state and ultimately concatinate tables across 4 states into a comprehensive database
```

Comment:
"use utils/pipeline.py" -- how? as a module? as a script? make it clear for new users (ie 'run `python utils/pipeline.py`)